test/packages/assets_elastic_versions_allowed_snapshot/manifest.yml

```diff
@@ -0,0 +1,35 @@
+format_version: 1.0.0
+name: assets_elastic_versions_allowed_snapshot
+title: "Assets Elastic versions allowed snapshot"
+version: 1.0.0
+source:
+  license: "Elastic-2.0"
+license: basic
+description: "Kibana assets exported using a Elastic SNAPSHOT version allowed from manifest"
+type: integration
+categories:
+  - custom
+conditions:
+  kibana.version: "^8.4.1"
+  elastic.subscription: "basic"
+screenshots:
+  - src: /img/sample-screenshot.png
+    title: Sample screenshot
+    size: 600x600
+    type: image/png
+icons:
+  - src: /img/sample-logo.svg
+    title: Sample logo
+    size: 32x32
+    type: image/svg+xml
+policy_templates:
+  - name: sample
+    title: Sample logs
+    description: Collect sample logs
+    inputs:
+      - type: logfile
+        title: Collect sample logs from instances
+        description: Collecting sample logs
+allow_snapshot: true
```

Comment:
Package that has set `allow_snapshot` as true. This package is allowed to contain assets built from Elastic stack SNAPSHOT versions